### PR TITLE
Allow command entry in Podman RuntimeConfig to overrides image's entrypoint - preparations

### PR DESCRIPTION
**Pull request by maturar**
Issues: #24 

# Definition of Done

The PR shall be merged **only if all of following items were checked.** In case an item is not applicable as described, please provide a short explanation right after the item in question. Separate the explanation by a semicolon and write it as bold text like **; requirements are handled in issue #xyz**:

- [x] documentation of all modules `/*/doc/swdesign` is up-to-date
- [x] requirements are up-to-date and requirements for new features have been added and mapped to source and test - This will be fixed in the issue #1 
- [x] conform to [unit verification strategy](https://eclipse-ankaios.github.io/ankaios/development/unit-verification/) - This will be fixed in the issue #1

### CODE REVIEW

File: agent/src/podman/workload.rs [hunk 1]
```diff
@@ -18,94 +18,26 @@ use common::state_change_interface::StateChangeInterface;
 use common::state_change_interface::StateChangeSender;
 
 use hyper::http;
-use podman_api::models::{ContainerMount, PortMapping};
-use podman_api::opts::{ImageListFilter, ImageListOpts, PullOpts};
 use podman_api::Error;
```
Comment (krucod3) [inline agent/src/podman/workload.rs:21]: We should not have dependencies to podman_api here too.

**Review by krucod3 — COMMENTED** [review]
Some additional dependencies need to be removed. As I did not know if it would work or not I tried it out and directly committed the changes to the branch.
Please have a look and let me know what you think.

File: agent/src/podman/podman_runtime_config.rs [hunk 2]
```diff
@@ -0,0 +1,68 @@
+use std::collections::HashMap;
+
+use common::objects::WorkloadSpec;
+use podman_api::models::PortMapping;
```
[the remark this answers is not in the document]
Comment (maturar) [reply agent/src/podman/podman_runtime_config.rs]: Fixed

File: agent/src/podman/workload.rs (hunk 1 above)
Comment (maturar) [reply agent/src/podman/workload.rs]: Fixed

**Comment by maturar** [comment]
Changes from @krucod3 is okay for me.

**Review by krucod3 — APPROVED** [review]
LGTM 👍

**Pull request merged**